frontend/Views/ForumFeed/reducers.js

```diff
@@ -77,13 +78,24 @@ export const feedReducer = (state = initialState, action) => {
       return Object.assign({}, state, {
         sortingMethod: action.payload,
       });
-
+    
+    
+    case SEARCH:
+      const searchInput = action.payload;
+      const filteredDiscussions = state.discussions.filter((discussion) => discussion.title.toLowerCase().includes(searchInput.toLowerCase()));
```

Comment:
Should we search on the body of the discussion too? (for example, run the filter through the opinions)? I think this is fine for now but maybe create and issue or TODO for this?

Reply:
I could make a new issue for that.  How do you feel about 3 checkboxes: title, body , and tags to use to filter the searches? And have all three show up as default?

Reply:
Sounds good to me!